src/main/protocol.ts

```diff
@@ -92,6 +92,12 @@ export const listenForProtocolHandler = () => {
       handlePotentialProtocolLaunch(url);
     }
   });
+  app.on('open-file', (_, path) => {
+    if (!path || path.length < 1) {
+      return;
+    }
+    ipcMainManager.send(IpcEvents.FS_OPEN_FIDDLE, [path]);
+  });
```

Comment:
Why was this handler necessary here? I'm not very familiar with the `open-file` event.

Reply:
When you click on a file from either the menu or the context menu, a open-file event is [fired](https://www.electronjs.org/docs/tutorial/recent-documents). So we need this handler in order to do something with the event, in this case open a new fiddle w/ the specified path

otherwise nothing happens when you try to open a recent file

Reply:
^ the open-file tidbit is mentioned at the very bottom of the linked page